### PR TITLE
Extract version label chrome logic to a dedicated class

### PR DESCRIPTION
**Pull request by dragunoff**
Reduces some duplication and encapsulates this simple logic.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/VersionLabelLogic.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+using OpenRA.Widgets;
```
Comment (pchote) [inline OpenRA.Mods.Common/Widgets/Logic/VersionLabelLogic.cs:12]: Missing file header

**Review by pchote — COMMENTED** [review]
Good idea, this also solves some duplication i had in TDHD 🎉

**Comment by dragunoff** [comment]
Update: added file header.

**Review by PunkPun — APPROVED** [review]
LGTM

**Comment by PunkPun** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/cf0c594a019cf3fb058db0974fcf5ecde8b4e808...135c6460eb74fae49a2e8b3daa032b843549d0a1)

**Pull request merged**